fdbserver/workloads/HealthMetricsApi.actor.cpp

```diff
@@ -168,6 +168,7 @@ struct HealthMetricsApiWorkload : TestWorkload {
 				traceDiskUsage.detail(format("Storage-%s", ss.first.toString().c_str()), storageStats.diskUsage);
 			}
 			TraceEvent traceTLogQueue("TLogQueue");
+			traceTLogQueue.setMaxFieldLength(10000).setMaxEventLength(11000);
```

Comment:
It looks like the fields here are all small, so this can just change to set the max event length:

```suggestion
			traceTLogQueue.setMaxEventLength(10000);
```